docs/sample-out/section=2385758954/attendance-events/2020-09-17-15-04-23.csv

```diff
@@ -1,3 +1,3 @@
 ﻿SourceSystemIdentifier,SourceSystem,EventDate,AttendanceStatus,LMSSectionSourceSystemIdentifier,LMSUserSourceSystemIdentifier,CreateDate,LastModifiedDate,SourceCreateDate,SourceLastModifiedDate
 2385758954-2020-08-28,BestLMS,2020-08-28 00:01:03,Excused,2385758954,100032890,2020-08-20 12:34:56,2020-08-20 12:34:56,,
-2385758954-2020-08-29,BestLMS,2020-08-28 00:01:03,Unexcused,2385758954,100032891,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+2385758955-2020-08-29,BestLMS,2020-08-28 00:01:03,Unexcused,2385758954,100032891,2020-08-20 12:34:56,2020-08-20 12:34:56,,
```

Comment:
Isn't our standard to use the section identifier here? In which case the old value was correct.
```suggestion
2385758954-2020-08-29,BestLMS,2020-08-28 00:01:03,Unexcused,2385758954,100032891,2020-08-20 12:34:56,2020-08-20 12:34:56,,
```

Reply:
Well, it is actually more complicated than just the section identifier. In this file it doesn't really matter.